### PR TITLE
add kal-loader feature

### PR DESCRIPTION
**Pull request by ydomenjoud**

### CODE REVIEW

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loader.component.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+@Component({
+  selector: 'kal-loader',
+  template: `
+    <p> Chargement en cours </p>
+  `,
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class KalLoaderComponent implements OnInit {
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loader.component.ts]: Remove onInit

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loading.directive.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    if (loading === true) {
+      this.attachLoader();
+    } else if (this._loading === true) {
+      this.getOverlayRef().detach();
+    }
+    this._loading = loading;
+  }
+
+  private attachLoader() {
+    const portal = new ComponentPortal(KalLoaderComponent);
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loading.directive.ts]: this.getOverlayRef().attach(new ComponentPortal(KalLoaderComponent));

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loader.component.ts (hunk 1 above)
Comment (jsp-up) [reply projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loader.component.ts]: We should be able to change to loading message

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loader.component.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, ViewEncapsulation, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+  selector: 'kal-loader',
+  template: `
+    <p> Chargement en cours </p>
```
Comment (jsp-up) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loader.component.ts]: We should be able to change the loading message

**Review by jsp-up — CHANGES_REQUESTED** [review]

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loading.directive.ts (hunk 2 above)
Comment (ydomenjoud) [reply projects/kalidea/kaligraphi/src/lib/atoms/kal-loader/kal-loading.directive.ts]: this proposal reduce clarity

**Review by jsp-up — APPROVED** [review]

**Pull request merged**